Formula/vite.rb

```diff
@@ -3,8 +3,8 @@
 class Vite < Formula
   desc "Next generation frontend tooling. It's fast!"
   homepage "https://vitejs.dev/"
-  url "https://registry.npmjs.org/vite/-/vite-4.4.7.tgz"
```

Comment:
what is the rationale for this change?

Reply:
This was my mistake, I’ll revert this to the previous URL format when the NPM package is published.

Reply:
👍